documentation/assemblies/upgrading/assembly-downgrade-kafka-versions.adoc

```diff
@@ -1,20 +1,13 @@
 // This assembly is included in the following assemblies:
 //
-// assembly-upgrade.adoc
-
-
-[id='assembly-downgrading-kafka-versions-{context}']
+// assembly-downgrade.adoc
 
+[id='assembly-downgrade-kafka-versions-{context}']
 = Downgrading Kafka
 
 Kafka version downgrades are performed using the Cluster Operator.
 
-Whether and how the Cluster Operator performs a downgrade depends on the differences between versions of:
-
-* Interbroker protocol
-* Log message format
-* ZooKeeper
-
+//Version constraints on the downgrade
```

Comment:
Should this still be here?

Reply:
Note on the file included in the assembly